src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="quickAdd_container" id="workItemQuickAdd_container" >
+<div class="quickAdd_container" id="workItemQuickAdd_container" *ngIf="wilistview === 'wi-list-view'" >
```

Comment:
Let them be like this for now in this line, because they are not part of this PR.